app/src/main/java/ch/hikemate/app/MainActivity.kt

```diff
@@ -86,6 +89,7 @@ fun HikeMateApp() {
           FirebaseAuthRepository(), ProfileRepositoryFirestore(FirebaseFirestore.getInstance()))
 
   val hikesViewModel: HikesViewModel = viewModel(factory = HikesViewModel.Factory)
+  val facilitiesViewModel = FacilitiesViewModel(FacilitiesRepositoryOverpass(OkHttpClient()))
```

Comment:
Warning: It seems that the author of the FacilitiesVM (me...) didn't implement a factory, as done with the other vMs. I dont remember exactly why, but there was a clear benefit to doing this. Do you mind adding this to the vM and using that instead? Afaik chatgpt or claude are good at this stuff, or you could just look at e.g. the HikesViewModel.factory

Reply:
Done